modopt/opt/proximity.py

```diff
@@ -475,8 +475,9 @@ def __init__(self, weights):
                               ' found see documentation for details: '
                               'https://cea-cosmic.github.io/ModOpt/'
                               '#optional-packages')
-
-        self.weights = np.sort(weights.flatten())[::-1]
+        if np.max(np.diff(weights)) > 0:
+            raise ValueError('Weights must be non increasing')
```

Comment:
As a user, I am not sure would understand what `'Weights must be non increasing'` means. Perhaps you can rephrase this. *e.g.* "the weight values must be provided in descending order" or something like that.

Reply:
Done